javaparser-core/src/main/java/com/github/javaparser/ast/StubUnit.java

```diff
@@ -0,0 +1,153 @@
+package com.github.javaparser.ast;
+
+import com.github.javaparser.TokenRange;
+import com.github.javaparser.ast.visitor.GenericVisitor;
+import com.github.javaparser.ast.visitor.VoidVisitor;
+import com.github.javaparser.metamodel.InternalProperty;
+import com.github.javaparser.printer.PrettyPrinter;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+import static com.github.javaparser.Providers.UTF8;
+
+/**
+ * <p>
+ * This class represents a stub file. The stub file is a Java file, but with the optionally omitted
+ * information that is not relevant to pluggable type-checking; this makes the stub file smaller
+ * and easier for people to read and write.
+ * </p>
+ * A stub unit contains the list of compilation units.
+ * This class copied the {@link CompilationUnit} and then adjusted to the needs of the Checker Framework.
+ *
+ * @see CompilationUnit
+ */
+public class StubUnit extends Node {
+
+    /**
+     * The field represents a list of compilations units.
```

Comment:
The field doesn't represent a list of compilation units; it *is* a list of compilation units. Make the comment give information beyond the type if possible.

(Typo: "compilations units" should be "compilation unit".)